packages/compass-connections-navigation/src/connections-navigation-tree.tsx

```diff
@@ -112,7 +117,8 @@ const ConnectionsNavigationTree: React.FunctionComponent<
             item.connectionInfo?.savedConnectionType === 'favorite';
           if (item.connectionStatus === ConnectionStatus.Connected) {
             return connectedConnectionItemActions({
-              isReadOnly: preferencesReadOnly || item.isConnectionReadOnly,
+              isReadOnly: item.isReadOnly,
```

Comment:
NIT: as you have probably noticed "readonly" is overloaded with meanings, and even if here is attached to item, probably is better to use a longer more descriptive name that reveals its intent?

What is this flag controlling in this case?

Reply:
This is only to filter out the "write" options from the popover menu, right? So maybe a var name or set of var names that unambiguously reflects that?

Reply:
@lerouxb what do you mean by var name?

Reply:
does `areWriteActionsEnabled` sound better?